### PR TITLE
Unsupported storage & network drivers feature flag

### PR DESCRIPTION
**Pull request by vincent99**
For UI feature to toggle showing storage drivers that are not commercially supported

### CODE REVIEW

**Comment by rmweir** [comment]
Is the intent of this just to start the feature flag? Are you going to check its value via API? It's not attached to a schema or anything.

**Comment by vincent99** [comment]
They are going to continue to be visible and usable in the sky in this case, but hidden from the UI unless you go toggle the feature because customers are confused about which drivers are commercially supported.

We will build out a UI to toggle the features later.

**Comment by rmweir** [comment]
> They are going to continue to be visible and usable in the sky in this case, but hidden from the UI unless you go toggle the feature because customers are confused about which drivers are commercially supported.
> 
> We will build out a UI to toggle the features later.

okay, lgtm

File: pkg/features/feature.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ var (
 	features = make(map[string]*feature)
 
 	// Features, ex.: ClusterRandomName = newFeature("cluster-randomizer", false)
```
Comment (rmweir) [inline pkg/features/feature.go]: just need a newline here

**Comment by codyrancher** [comment]
Did you verify the return value of the feature flag? When I applied this change locally the name and id fields were as expected but the value field was null.

**Comment by cjellick** [comment]
https://github.com/rancher/rancher/issues/22895

**Pull request merged**